models/br_anatel_banda_larga_fixa/schema.yml

```diff
@@ -0,0 +1,67 @@
+version: 2
+
+models:
+  - name: microdados
+    description: Microdados de banda larga fixa (também conhecido como Serviço de Comunicação Multimídia) a partir de 2007 que traz dados mensais acerca da empresa ofertante e das especificações técnicas do serviço.
```

Comment:
aqui acho que a descrição é p/ o modelo. melhor colocar algo como "dbt model for table <table_id>", como está em outros conjuntos.

Reply:
Alterei a descrição do modelo, Arthur. Muito obrigado pela observação.